apps/signer/package.json

```diff
@@ -22,14 +22,14 @@
     "test:integration:debugdb": "VERBOSE_DB_LOGGING=true jest --runInBand test/integration",
     "test:e2e": "jest --runInBand test/end-to-end --testPathIgnorePatterns test/end-to-end/disabled-apis.test.ts",
     "test:e2e:disabledapis": "jest --runInBand test/end-to-end/disabled-apis.test.ts",
-    "test:e2e:staging:0": "CONTEXT_NAME=staging ODIS_SIGNER_SERVICE_URL=https://staging-pgpnp-signer0.azurefd.net yarn test:e2e",
-    "test:e2e:staging:1": "CONTEXT_NAME=staging ODIS_SIGNER_SERVICE_URL=https://staging-pgpnp-signer1.azurefd.net yarn test:e2e",
-    "test:e2e:staging:2": "CONTEXT_NAME=staging ODIS_SIGNER_SERVICE_URL=https://staging-pgpnp-signer2.azurefd.net yarn test:e2e",
-    "test:e2e:alfajores:1": "CONTEXT_NAME=alfajores ODIS_SIGNER_SERVICE_URL=https://odis-alfajores-signer-1-b.azurefd.net yarn test:e2e",
-    "test:e2e:alfajores:2": "CONTEXT_NAME=alfajores ODIS_SIGNER_SERVICE_URL=https://odis-alfajores-signer2.azurefd.net yarn test:e2e",
-    "test:e2e:alfajores:3": "CONTEXT_NAME=alfajores ODIS_SIGNER_SERVICE_URL=https://odis-alfajores-signer3.azurefd.net yarn test:e2e",
-    "test:e2e:mainnet:brazilsouth": "CONTEXT_NAME=mainnet ODIS_SIGNER_SERVICE_URL=https://mainnet-pgpnp-brazilsouth.azurefd.net yarn test:e2e",
-    "test:e2e:mainnet:eastasia": "CONTEXT_NAME=mainnet ODIS_SIGNER_SERVICE_URL=https://mainnet-pgpnp-eastasia.azurefd.net yarn test:e2e",
+    "test:e2e:staging:0": "CONTEXT_NAME=staging ODIS_SIGNER_SERVICE_URL=http://odis-signer0-staging.odis-signer0-staging:3000 yarn test:e2e",
+    "test:e2e:staging:1": "CONTEXT_NAME=staging ODIS_SIGNER_SERVICE_URL=http://odis-signer1-staging.odis-signer1-staging:3000 yarn test:e2e",
+    "test:e2e:staging:2": "CONTEXT_NAME=staging ODIS_SIGNER_SERVICE_URL=http://odis-signer2-staging.odis-signer2-staging:3000 yarn test:e2e",
+    "test:e2e:alfajores:1": "CONTEXT_NAME=alfajores ODIS_SIGNER_SERVICE_URL=http://odis-signer0-alfajores.odis-signer0-alfajores:3000 yarn test:e2e",
+    "test:e2e:alfajores:2": "CONTEXT_NAME=alfajores ODIS_SIGNER_SERVICE_URL=http://odis-signer1-staging.odis-signer1-staging:3000 yarn test:e2e",
+    "test:e2e:alfajores:3": "CONTEXT_NAME=alfajores ODIS_SIGNER_SERVICE_URL=http://odis-signer2-alfajores.odis-signer2-alfajores:3000 yarn test:e2e",
+    "test:e2e:mainnet:brazilsouth": "CONTEXT_NAME=mainnet ODIS_SIGNER_SERVICE_URL=http://odis-signer0-mainnet.odis-signer0-mainnet:3000 yarn test:e2e",
```

Comment:
@alvarof2 just want to confirm that the `brazilsouth` && `eastasia` have been migrated to `odis-signer0-mainnet.odis-signer0-mainnet` && `odis-signer1-mainnet.odis-signer1-mainnet` respectively.